app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +141,106 @@ export const CollectionsPage = ({ data }) => {
           subtitle="Explore the New York Public Library's diverse collections, including digitized photographs, manuscripts, maps, and more. Start exploring by using the search bar below or browse through the collections."
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
           id={"search-collections"}
           textInputProps={{
             isClearable: true,
             labelText: "Search by collection title",
-            name: "searchinput",
+            name: "collection_keyword",
             placeholder: "Search by collection title",
+            defaultValue: currentCollectionKeyword,
+            onChange: (e) => handleSearchChange(e),
           }}
-          onSubmit={function (event: React.FormEvent): void {}}
+          onSubmit={handleSearchSubmit}
           labelText={""}
         />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <SearchResults showFilter isSearchPage={false} data={data} />
+
+      {isLoaded ? (
+        collections.length > 0 ? (
+          <>
+            <Box sx={{ display: "flex", gap: "xs", marginBottom: "l" }}>
+              <Text
+                sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}
+              >
+                {" "}
+                Sort by{" "}
+              </Text>{" "}
+              <Menu
+                showSelectionAsLabel
+                showLabel
+                selectedItem={currentSort}
+                labelText={"Sort By"}
+                listItemsData={[
+                  {
+                    id: "date-desc",
+                    label: "Newest to oldest",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "date-asc",
+                    label: "Oldest to newest",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "title-asc",
+                    label: "Title A to Z",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "title-desc",
+                    label: "Title Z to A",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                ]}
+              />
+            </Box>
+            <Heading
+              size="heading5"
+              sx={{ marginBottom: "l" }}
+              ref={headingRef}
```

Comment:
By putting the `Menu` and this header in the conditional, when you perform a search or sort, focus gets lost. Focus should go to the heading after performing a search or selecting a sorting option.